app/models/manageiq/providers/openstack/storage_manager/cinder_manager/cloud_volume.rb

```diff
@@ -85,15 +85,14 @@ def self.params_for_create(ems)
   end
 
   def self.raw_create_volume(ext_management_system, options)
+    options.symbolize_keys!
```

Comment:
My only concern here is that we are modifying the caller's options...not sure we should do that or not. Similarly on raw_update_volume

Reply:
```suggestion
    options = options.dup.symbolize_keys!
```

Reply:
> My only concern here is that we are modifying the caller's options

Yeah that's fair

>     options = options.dup.symbolize_keys!

well don't need the dup and the bang version of symbolize_keys!, can just `options = options.symbolize_keys`